packages/utils/src/revert_error.ts

```diff
@@ -113,7 +113,9 @@ export abstract class RevertError extends Error {
         const instance = new type();
         try {
             const values = decoder(_bytes);
-            return _.assign(instance, { values });
+            _.assign(instance, { values });
+            instance.message = instance.toString();
```

Comment:
This allows revert errors to finally print their full payloads in the stack traces. You're welcome.